test-framework/src/main/java/org/apache/brooklyn/test/framework/RelativeEntityTestCase.java

```diff
@@ -52,12 +51,14 @@
 @ImplementedBy(RelativeEntityTestCaseImpl.class)
 public interface RelativeEntityTestCase extends TargetableTestComponent {
 
+    @SetFromFlag("anchor")
     AttributeSensorAndConfigKey<Entity, Entity> ANCHOR = ConfigKeys.newSensorAndConfigKey(Entity.class,
-            "anchor",
+            "test.anchor.entity",
```

Comment:
I don't think we should rename all these. The semantics for SetFromFlag is different (for inheritance) so this is not backwards compatible. I think we should leave as-is. The old name was fine. We don't need to make the key names "slightly unique".